src/config/dynamic-theme-fixes.config

```diff
@@ -7684,6 +7684,19 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+cursoreffects.com
+
+CSS
+.box {
+    background-color: #808080;
+    border-bottom-color: var(--darkreader-border-000000, #FF1694);
```

Comment:
If the value should be strictly `#FF1694`, please use just `#FF1694 !important` without referencing Dark Reader internal variables. `--darkreader-border-000000` refers to a color, that used to be black initially.